lib/jsonschema-tools.js

```diff
@@ -135,7 +135,7 @@ const jsonSchemaFakerOptions = {
     // By setting this to a constant value, json-schema-faker
     // will always return the same example value for the
     // same type of field.
-    random: () => 0.15
+    random: () => 9
```

Comment:
Would you mind explaining a bit more about the problem and how this solves it?

It looks like the `random` option is replacing the default `Math.random` function, which returns a floating-point number in [0, 1).  A number greater than or equal to 1 will cause unexpected results.

If there's a problem with overriding randomness, maybe it's caused by something other than this lamba?

Reply:
I'm sorry. I seems like I also got it wrong, it should be 4:

https://www.explainxkcd.com/wiki/index.php/221:_Random_Number